utils/src/main/scala/com/salesforce/op/test/TestSparkContext.scala

```diff
@@ -60,22 +59,11 @@ trait TestSparkContext extends TempDirectoryTest with TestCommon {
   implicit lazy val spark: SparkSession = SparkSession.builder.config(conf).getOrCreate()
   implicit lazy val sc: SparkContext = spark.sparkContext
 
-  lazy val checkpointDir: String = createDirectory(tempDir.getCanonicalPath, "checkpoints").toString
-
   override def beforeAll: Unit = {
     super[TempDirectoryTest].beforeAll()
-    sc.setCheckpointDir(checkpointDir)
+    val _ = sc // triggers initialization of spark session
   }
 
-  override def afterAll: Unit = {
```

Comment:
Does it mean that we no longer want to cleanup temp directories created by spark?

Reply:
It’s being cleared in the base trait